Builds/VisualStudio2017/README.md

```diff
@@ -0,0 +1,260 @@
+# Visual Studio 2017 Build Instructions
+
+## Important
+
+We do not recommend Windows for rippled production use at this time. Currently,
+the Ubuntu platform has received the highest level of quality assurance,
+testing, and support. Additionally, 32-bit Windows versions are not supported.
+
+## Prerequisites
+
+To clone the source code repository, create branches for inspection or
+modification, build rippled under Visual Studio, and run the unit tests you will
+need these software components:
+
+* [Visual Studio 2017](README.md#install-visual-studio-2017)
+* [Git for Windows](README.md#install-git-for-windows)
+* [Google Protocol Buffers
+  Compiler](README.md#install-google-protocol-buffers-compiler)
+* [OpenSSL Library](README.md#install-openssl)
+* [Boost library](README.md#build-boost)
+* (Optional) [Cmake for Windows](README.md#optional-install-cmake-for-windows) -
+  Only needed if not using the integrated CMake in VS 2017.
+
+## Install Software
+
+### Install Visual Studio 2017
+
+If not already installed on your system, download your choice of installer from
+the [Visual Studio 2017
+Download](https://www.visualstudio.com/downloads/download-visual-studio-vs)
+page, run the installer, and follow the directions. **You may need to choose the
+`Desktop development with C++` workload to install all necessary C++ features.**
+
+Any version of Visual Studio 2017 may be used to build rippled. The **Visual
+Studio 2017 Community** edition is available free of charge (see [the product
+page](https://www.visualstudio.com/products/visual-studio-community-vs) for
+licensing details), while paid editions may be used for an free initial trial
+period.
+
+As of this writing, the latest version of Visual Studio 2017 for Windows is
```

Comment:
I don't object to this being here, but also don't think there is any benefit to mentioning current latest versions of software (one of them will surely be eclipsed within a short time). What *could* be useful is a table of known to work versions for each dependency. Perhaps this is what you were getting at here - if that's the case, I'd consider just making a table of each prerequisite and the version(s) that are known to work.